tamr_toolbox/utils/recipe.py

```diff
@@ -0,0 +1,35 @@
+"""Tasks related to project recipe"""
+import logging
+
+from tamr_unify_client.mastering.project import MasteringProject
+
+from tamr_toolbox.models.project_type import ProjectType
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_dedup_recipe_id(project: MasteringProject) -> int:
+    """
+    Retrieves dedup recipe ID of a mastering project.
+    To be used to run custom recipe operations.
+
+    Args:
+        project: a mastering project
+
+    Returns: dedup recipe id of the project.
+
+    """
+    # Retrieve project steps using unified dataset
+    project_steps = project.unified_dataset().usage().usage.input_to_project_steps
+    # Search for dedup project step of the mastering project
+    for step in project_steps:
+        if step.project_name == project.name and step.type == ProjectType.DEDUP.value:
```

Comment:
This breaks if the project name is manually changed after creating the project, is there a reason for this part? 
```suggestion
        if step.type == ProjectType.DEDUP.value:
```